docs/src/modules/spring/pages/access-control.adoc

```diff
@@ -1 +1,201 @@
+
+The Maven archetype include a less permissive ACL for the entire service, to not accidentally make services available to the public internet, just like the one described in the next section.
+
+=== Customizing the deny code
+
+When a request is denied, by default, a gRPC error code of 7, `PERMISSION_DENIED`, is sent. This gets transcoded to an HTTP status code of 403, forbidden. The code that is returned when a request is denied can be customised using the `deny_code` property. The deny code must be a valid gRPC error code, which is any integer from 1 to 16, described https://grpc.github.io/grpc/core/md_doc_statuscodes.html[here]. Each one maps as follows to an https://doc.akka.io/japi/akka-http/current/akka/http/scaladsl/model/StatusCodes.html[Akka HTTP status code].
+
+```
+    Status.Code.OK                  => StatusCodes.OK
+    Status.Code.CANCELLED           => StatusCodes.InternalServerError
+    Status.Code.UNKNOWN             => StatusCodes.InternalServerError
+    Status.Code.INVALID_ARGUMENT    => StatusCodes.BadRequest
+    Status.Code.DEADLINE_EXCEEDED   => StatusCodes.GatewayTimeout
+    Status.Code.NOT_FOUND           => StatusCodes.NotFound
+    Status.Code.ALREADY_EXISTS      => StatusCodes.Conflict
+    Status.Code.PERMISSION_DENIED   => StatusCodes.Forbidden
+    Status.Code.RESOURCE_EXHAUSTED  => StatusCodes.EnhanceYourCalm
+    Status.Code.FAILED_PRECONDITION => StatusCodes.BadRequest
+    Status.Code.ABORTED             => StatusCodes.InternalServerError
+    Status.Code.OUT_OF_RANGE        => StatusCodes.BadRequest
+    Status.Code.UNIMPLEMENTED       => StatusCodes.NotFound // Note: not the same as HTTP NotImplemented
+    Status.Code.INTERNAL            => StatusCodes.InternalServerError
+    Status.Code.UNAVAILABLE         => StatusCodes.ServiceUnavailable
+    Status.Code.DATA_LOSS           => StatusCodes.InternalServerError
+    Status.Code.UNAUTHENTICATED     => StatusCodes.Forbidden
+
+```
+
+For example, to make Kalix reply with 404, `NOT_FOUND`:
+
+
+[source, java]
+----
+include::example$spring-acl-examples/src/main/java/com/example/MyAction.java[tag=acl6]
+----
+
+Deny codes, if not specified on an ACL, are inherited from the service, or the default, so updating the `deny_code` in the default ACL policy will set it for all methods:
+
+
+[source, java]
+----
+include::example$spring-acl-examples/src/main/java/com/example/UserAction.java[tag=acl]
+----
+
+== ACLs on eventing methods
+
+
+Any method with an `@Subscribe` annotation on it will not automatically inherit either the default or its component's ACL, rather, all outside communication will be blocked, since it's assumed that a method that subscribes to an event stream must only be intended to be invoked in response to events on that stream. This can be overridden by explicitly defining an ACL on that method:
```

Comment:
Missing example here or wrong ponctuation? since it says: `by explicitly defining an ACL on that method:`

Reply:
Good catch. Added example.